docs/tutorials/proposals/asset-transfer-proposal.md

```diff
@@ -0,0 +1,195 @@
+* **Closing timestamp**, the date and time when voting closes, must be within a range of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.minClose" hideName={true}/> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxClose" hideName={true}/>, from when it was proposed.
+* **Enactment timestamp**, the date and time when the transfer is to happen, must be within a range of <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.minEnact" hideName={true}/> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxClose" hideName={true}/>.
+
+**Amount** is how much to transfer, as whole number with the asset decimal places implied. For example, if the asset has a decimal place of 2 and the transfer is for 100, then the amount needs to be set at 10000. The maximum you can propose to transfer is governance.proposal.transfer.maxAmount. Before proposing, make sure the account you're transferring from exists and has a balance. The full amount may not be transferred if there isn't enough to transfer.
+
+**Fraction of balance** is used to determine what fraction of the source account's balance could be transferred, the biggest fraction that you can choose is <NetworkParameter frontMatter={frontMatter} param="governance.proposal.transfer.maxFraction" hideName={true}/>.
+
+**Governance transfer type** lets you determine if you want to try to transfer all or nothing, or to transfer as much as is possible up to the full amount.
+
+### Fields used to fund trading rewards
+
+If you're proposing a transfer to fund rewards, there are extra fields to define.
+
+The `toAccountType` must be the account type that matches the reward category. For example, to propose that the 'average position' reward will pay out, you'll need to set the "to account type" type as `ACCOUNT_TYPE_REWARD_AVERAGE_POSITION`, and then choose the complementary reward category, known as the `metric`. The asset you choose then determines which market(s) the reward targets.
+
+The `to` field is used for public keys. In the case of network-managed accounts, the input they need is `0000000000000000000000000000000000000000000000000000000000000000`.
+
+You will need to define the dispatch strategy, which includes the  metric, the length of time to measure performance, the asset used to evaluate performance, and other fields. 
+
+| Dispatch strategy field | Description | Accepted values |
+| ----------- | ----------- | ----------- |
+| `assetForMetric` | Asset that's used to evaluate how someone performs, such as the settlement asset for the market(s) relevant to the reward | Any asset enabled on Vega |
```

Comment:
not sure what you mean by "such as the settlement asset", it's literally the settlement asset for the markets to be included, it can't be something else.